install/kubernetes/cilium/templates/validate.yaml

```diff
@@ -167,9 +167,6 @@
 {{- end -}}
 
 {{- if and .Values.clustermesh.useAPIServer (eq .Values.clustermesh.apiserver.kvstoremesh.kvstoreMode "internal") }}
-  {{- if and (ne .Values.identityAllocationMode "crd") (ne .Values.identityAllocationMode "doublewrite-readkvstore") (ne .Values.identityAllocationMode "doublewrite-readcrd") }}
-    {{ fail (printf "The clustermesh-apiserver cannot be enabled in combination with .Values.identityAllocationMode=%s. To establish a Cluster Mesh, directly configure the parameters to access the remote kvstore through .Values.clustermesh.config" .Values.identityAllocationMode ) }}
-  {{- end }}
   {{- if .Values.disableEndpointCRD }}
     {{ fail "The clustermesh-apiserver cannot be enabled in combination with .Values.disableEndpointCRD=true" }}
   {{- end }}
```

Comment:
This check should be relaxed as well, so that `disableEndpointCRD` is supported in kvstore mode.